crates/swc_ecma_minifier/src/compress/pure/evaluate.rs

```diff
@@ -368,7 +368,7 @@ impl Pure<'_> {
                 } else {
                     // 11. Else,
 
-                    if x.fract() != 0. && f != 0 {
```

Comment:
@magic-akari I did this because of the regression, and I'm not sure about the correct fix

Reply:
Ok, I will check this soon.